packages/playground/ssr-framework/vue-framework/build.js

```diff
@@ -0,0 +1,37 @@
+const { getViteConfig } = require('./getViteConfig')
+const hydrateFile = require.resolve('./hydrate')
+const getPagesFile = require.resolve('./getPages')
+
+module.exports.build = build
```

Comment:
**suggestion (dx):** Just a style choice, but could we first define build and then use it? :slightly_smiling_face: 
So just move this line to the bottom of the file